.github/workflows/update-helm-chart.yml

```diff
@@ -0,0 +1,138 @@
+            echo "::set-output name=helm_older::$HELM_OLDER"
+          fi
+
+      - name: Compare Helm Versions
+        id: compare-versions
+        run: |
+          if [ "$HELM_NEWER" != "$HELM_OLDER" ]; then
+            echo "New version available: $HELM_NEWER"
+            echo "::set-output name=new_version::true"
+          else
+            echo "No new version available"
+            echo "::set-output name=new_version::false"
+          fi
+
+  create-issue:
+    name: Create Issue on Upgrade Failure
+    needs: upgrade-helm-chart
+    runs-on: ubuntu-22.04
+    if: needs.upgrade-helm-chart.outputs.upgrade_failed == 'true'
+    steps:
+      - name: Create Issue
+        run: |
+          echo "Creating issue..."
+          ACCESS_TOKEN="${{ secrets.GITHUB_TOKEN }}"
+          REPO_OWNER="keptn"
+          REPO_NAME="lifecycle-toolkit"
+          ISSUE_TITLE="${{ needs.upgrade-helm-chart.outputs.upgrade_error_message }}"
+          ISSUE_BODY="The Helm upgrade for the keptn chart has failed with the following error message:\n\n\
+          `${{ needs.upgrade-helm-chart.outputs.upgrade_error_message }}\`"
+
+          curl -X POST -H "Authorization: token $ACCESS_TOKEN" \
+               -H "Accept: application/vnd.github.v3+json" \
+               https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues \
+               -d "{\"title\":\"$ISSUE_TITLE\",\"body\":\"$ISSUE_BODY\"}"
+
+  run-integration-test:
+    needs: upgrade-helm-chart
+    name: Run Chainsaw Tests
+    runs-on: ubuntu-22.04
+    if: needs.upgrade-helm-chart.outputs.new_version == 'true'
+    strategy:
+      matrix:
+        helm-chart-version:
+          [
+            "${{ needs.upgrade-helm-chart.outputs.helm_newer }}",
+            "${{ needs.upgrade-helm-chart.outputs.helm_older }}",
+          ]
+    steps:
+      - name: Install and expose Prometheus
+        uses: ./.github/actions/deploy-prometheus-on-cluster
+
+      - name: Install Chainsaw
+        uses: kyverno/action-install-chainsaw@v0.1.7
+
+      - name: Run Scheduling Gates Integration Tests
+        if: inputs.scheduling-gates == 'gates_on' && inputs.allowed-namespaces == 'allowed_ns_off'
+        working-directory: .
+        run: make integration-test-scheduling-gates
+
+      - name: Run Allowed namespaces Integration Tests
+        if: inputs.allowed-namespaces == 'allowed_ns_on'
+        working-directory: .
+        run: make integration-test-allowed-namespaces
+
+      - name: Run Integration Tests
+        if: inputs.allowed-namespaces == 'allowed_ns_off'
+        working-directory: .
+        run: make integration-test
+
+      - name: Create reports
+        if: failure()
+        working-directory: ./.github/scripts
+        run: ./create-reports-full.sh
+
+      - name: Upload cluster logs
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: logs-integration-tests-${{ inputs.scheduling-gates }}-${{ inputs.allowed-namespaces }}
```

Comment:
We should not name it the same as the integration tests, otherwise it will be overwritten